modules/geojson-modes/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@nebula.gl/geojson-modes",
+  "description": "GeoJSON editing modes for nebula.gl",
+  "license": "MIT",
+  "version": "0.12.1",
+  "author": "Georgios Karnas <georgios@uber.com>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/uber/nebula.gl"
+  },
+  "keywords": [
+    "webgl",
+    "visualization",
+    "overlay",
+    "layer"
+  ],
+  "main": "dist/index.js",
```

Comment:
Info: For other repos We have moved to one dist folder. `dist/es5` `dist/esm` `dist/es6` `dist/script.min.js`...

Reply:
We can look at making that change for all the modules in a separate PR